content/graphing/infrastructure/process.md

```diff
@@ -23,24 +23,37 @@ Datadog Process Monitoring allows for real-time visibility of the most granular
 
 The following installation processes are for [agent v6 only](/agent), if you are still using agent v5, [follow this specific installation process](/agent/faq/agent-5-process-collection)
 
+## Process agent
+
 **Note**: Live Processes beta for Windows requires Agent 6.
 
 ### Standard Agent Configuration
- 
-Refer to the instructions for standard [Agent installation](https://app.datadoghq.com/account/settings#agent) for platform-specific details.
+
+The process agent is shipped by default with the Agent 6 in the Linux packages only. Refer to the instructions for standard [Agent installation](https://app.datadoghq.com/account/settings#agent) for platform-specific details.
```

Comment:
I would remove the word `the` before `Agent 6`, and before `Linux`.

> The process agent is shipped by default with Agent 6 in Linux packages only.

Reply:
👌